package.json

```diff
@@ -14,10 +14,14 @@
     "node": "^12"
   },
   "scripts": {
-    "test": "true"
+    "test": "true",
+    "semrel-releaser": "lerna exec --scope=monorepo-releaser yarn semantic-release --stream",
+    "semrel-wildcards": "lerna exec --scope=semantic-release-wildcards yarn semantic-release --stream",
+    "semantic-release": "yarn semrel-releaser && yarn semrel-wildcards"
   },
+  "private": true,
```

Comment:
Doesn't it need to be a public repo for it to go to the public NPM?

Reply:
root repo is not released. Adding this was suggested in the docs for using `workspaces`. Should not affect the individual package releases